targets/targets.json

```diff
@@ -5271,6 +5271,60 @@
         "device_name": "STM32L433RC",
         "bootloader_supported": true
     },
+    "NUCLEO_L452RE_P": {
+        "inherits": [
+            "FAMILY_STM32"
+        ],
+        "supported_form_factors": [
+            "ARDUINO",
+            "MORPHO"
+        ],
+        "core": "Cortex-M4F",
+        "extra_labels_add": [
+            "STM32L4",
+            "STM32L452",
+            "STM32L452xx",
+            "STM32L452xE",
+            "STM32L452RE"
+        ],
+        "config": {
+            "clock_source": {
+                "help": "Mask value : USE_PLL_HSE_EXTC (need HW patch) | USE_PLL_HSE_XTAL (need HW patch) | USE_PLL_HSI | USE_PLL_MSI",
+                "value": "USE_PLL_MSI",
+                "macro_name": "CLOCK_SOURCE"
+            },
+            "lpticker_lptim": {
+                "help": "This target supports LPTIM. Set value 1 to use LPTIM for LPTICKER, or 0 to use RTC wakeup timer",
+                "value": 1
+            }
+        },
+        "macros_add": [
+            "STM32L452xx",
+            "STM32L452RE",
+            "MBED_TICKLESS",
+            "EXTRA_IDLE_STACK_REQUIRED",
+            "MBED_SPLIT_HEAP"
+        ],
+        "overrides": {
+            "lpticker_delay_ticks": 0
+        },
+        "detect_code": [
+            "0829"
+        ],
+        "device_has_add": [
+            "ANALOGOUT",
+            "CRC",
+            "SERIAL_ASYNCH",
+            "CAN",
+            "TRNG",
+            "FLASH",
+            "MPU"
+        ],
+        "release_versions": [
+            "5"
+        ],
+        "device_name": "STM32L452RE"
```

Comment:
Update value with "STM32L452RETx"
(value chosen in tools/arm_pack_manager/index.json)

Reply:
Thank you. I believe this resolved the build warning, as I also added `"bootloader_supported": true` and did not see any additional details in my latest test run.